Mobile menu

IshmaelKargbo opened the pull request:
In this branch we worked on:
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Please review and give any requested changes if required.

Thank you! 😄

IshmaelKargbo committed:
update

IshmaelKargbo committed:
update

IshmaelKargbo committed:
worked on mobile nav menu html & css

Shewolf227 committed:
Add JavaScript implementation

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

prolajumokeoni commented on index.html:
```diff
@@ -388,6 +380,21 @@ <h2 class="title">Contact me</h2>
+        <nav>
+            <ul id="nav-list" >
+                <li><a href="#headline">Portoflio</a></li>
+                <li><a href="#about">About</a></li>
```
- [x] Kindly add the about id to the right section in your page, currently when a user clicks it shows the works section 👍🏿

IshmaelKargbo replied:
Thanks 👍🏿

prolajumokeoni commented on index.html:
```diff
@@ -388,6 +380,21 @@ <h2 class="title">Contact me</h2>
         </footer>
     </main>
+    <!-- Menu Modal -->
+    <div id="menu-modal" class="menu-modal">
+        <nav>
+            <ul id="nav-list" >
+                <li><a href="#headline">Portoflio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+        <button id="close-modal" class="close-modal">
+            <span>&times;</span>
+        </button>
+    </div>
```
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_5](https://user-images.githubusercontent.com/69638013/161509772-8d06045b-1c2e-429a-a19f-5a13841832a9.png) |  ![Screenshot_6](https://user-images.githubusercontent.com/69638013/161516152-e4048cd7-5585-4a42-a6b3-b400f40cc6ab.png)

- [x] Kindly make the background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿 


- Adding this functionality will make your design good and look exactly like the Figma design, great job so far 💪🏿

IshmaelKargbo replied:
Thanks 👍🏿

prolajumokeoni reviewed (changes requested):
Hi @Team,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ GitHub flow was correctly used
✔️  Good design
✔️  Good PR and Description



## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

IshmaelKargbo committed:
Required Changes

[] Kindly add the about id to the right section in your page, currently when a user clicks it shows the works section 👍🏿
[] Kindly make the background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿

gemmen29 reviewed (approved):
Hi @IshmaelKargbo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
